static/css/marble.css

```diff
@@ -1667,6 +1638,19 @@ h6{
         -webkit-backdrop-filter: none;
     }
 
+    /*Home*/
+    .margin-usage-card{
```

Comment:
this class isn't used anywhere in the html

Reply:
It should be.  Added it to the html.